.github/scripts/test_cram_output.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -eo pipefail
+export PATH=/opt/conda/bin:$PATH
+
+echo test --outCram and --ref inputs >> artifacts/test_artifact.log
+
+export REF_FILE=$(find work_singularity_profile | grep -v scheme | grep 'ref.fa$' | xargs readlink | grep ncov2019-artic-nf | head -n1 | sed s'/ncov2019-artic-nf\/work\//ncov2019-artic-nf\/work_singularity_profile\//'g)
+echo ref file: $REF_FILE
+
+# run current pull request code
+# --sanger profile: there are only 2 available cpus in the github runner execution
+sed -i s'/cpus = 4/cpus = 2/'g conf/coguk/sanger.config
+singularity --version
+echo Nextflow run with --ref, mappedBam  >> artifacts/test_artifact.log
+NXF_VER=20.03.0-edge nextflow run $PWD/.github/workflows/test_bamToCram.nf \
+       -profile conda,sanger,singularity \
```

Comment:
You shouldn't need both conda and singularity profiles. Just 1 should work, and if not it's a bug.